### ISSUE TITLE
Global table search incorrectly applying to map

### ISSUE THREAD

**Issue by james-rae** [fgpv-vpgf/fgpv-vpgf#3543]
To replicate:  

- Run page `index-fgp-en.html?keys=EcoAction`
- Turn off all sublayers except for `Nature` (i.e. map should only show leaf icons)
- Open `Nature` table
- In global table search, type `bio`. Table filters to 145 records
- Click "Apply Filters to Map". Notice only one point appears.
- Using string `biol` results in no points on the map.

Looking at the query string on the web traffic, the field `Description` is getting included multiple times in the SQL WHERE clause.
(edit) would actually appear its adding a clause for every time the string is matched. Might need some field duplication checking in the global search to sql loop

**Comment by YaroRoodzit** [comment]
Tested build: v3.1.0-b1
The “EcoAction” layer has a problem in the current build. # 3580.Map points are not displayed for the “EcoAction” layer.

**Comment by YaroRoodzit** [comment]
Re- Tested build v3.1.0-b5
Same issue.

**Comment by james-rae** [comment]
Appears a new bug has been introduced between `v3.1.0-b4` and `v3.1.0-b5` that causes the map layer not to draw at all.

**Comment by james-rae** [comment]
Further investigation shows sporadic behavior with the map loading. We made a new rcs key `EcoActionProd` that uses the production version instead of the dev server, and saw same behavior.  Testing on our devices using outside internet and the map does what it is supposed to.

We have noticed very poor internet (slow, frequent request failures) in the past week.  Will put a pin in this for now, and will revisit when either the internet resumes normal service levels or v3.1.0 release date becomes imminent
